cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -91,6 +99,44 @@ bundle common cfe_internal_hub_vars
 
 }
 
+##################################################################
+#
+# update_cli_rest_server_url_config
+#  - updates REST server URL port of Mission Portal WebGUI when
+#    cfe_internal_hub_vars.https_port is changed
+#
+##################################################################
+bundle agent update_cli_rest_server_url_config
+{
+  vars:
+    "mp_config_file" string => "$(cfe_internal_hub_vars.docroot)/application/config/config.php";
```

Comment:
```suggestion
    # Both share and live versions must be changed at once since httpd will be restarted later in the same agent run.
    "mp_config_file" string => "$(cfe_internal_hub_vars.docroot)/application/config/config.php";
```